src/transformers/models/decision_transformer/configuration_decision_transformer.py

```diff
@@ -0,0 +1,205 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" DecisionTransformer model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+DECISION_TRANSFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "decision_transformer": "https://huggingface.co/decision_transformer/resolve/main/config.json",
```

Comment:
That url and checkpoint name can't work as they are missing an organization.

Reply:
Is this not the same for other models in the transformers library, such as [gpt2](https://github.com/huggingface/transformers/blob/master/src/transformers/models/gpt2/configuration_gpt2.py#L30) ?

Reply:
```suggestion
    "decision_transformer": "https://huggingface.co/edbeeching/decision-transformer-gym-hopper-medium/resolve/main/config.json",
```

Reply:
This needs to be addressed before merging. Patrick's suggestion is still missing the namespace in the key, which should be `edbeeching/decision-transformer-gym-hopper-medium` to match the model.